crates/egui/src/style.rs

```diff
@@ -1324,6 +1331,8 @@ impl Visuals {
             indent_has_left_vline,
 
             striped,
+
+            slider_trailing_fill: _,
```

Comment:
Please add a checkbox for it!

Reply:
Ah, I was wondering what this was. I've added one to the `Slider` demo as well.